server/src/action/manifest/create.js

```diff
@@ -0,0 +1,27 @@
+const AbstractAction = require('../abstract')
+const { body } = require('express-validator/check')
+const { sanitizeBody } = require('express-validator/filter')
+const controller = require('../../controller/manifest')
+
+class CreateManifest extends AbstractAction {
+  get validate () {
+    return [
+      sanitizeBody('name').trim(),
+      sanitizeBody('content').trim(),
```

Comment:
The manifest's content is trimmed on creation but not on update (see below). Is it intended, or is it a miss?